cider/type/plan/Expr.h

```diff
@@ -181,20 +181,27 @@ class Expr : public std::enable_shared_from_this<Expr> {
 
   JITExprValue* get_expr_value() const { return expr_var_.get(); }
 
+  void set_nulls(cider::jitlib::JITValuePointer& val) { nulls_.push_back(val); }
+  std::vector<cider::jitlib::JITValuePointer>& get_nulls() { return nulls_; }
+
+  void set_datas(cider::jitlib::JITValuePointer& val) { vals_.push_back(val); }
+  std::vector<cider::jitlib::JITValuePointer>& get_datas() { return vals_; }
+
   // TODO (bigPYJ1151): to pure virtual.
   virtual ExprPtrRefVector get_children_reference() {
     UNREACHABLE();
     return {};
   }
 
- protected:
   JITTypeTag getJITTag(const SQLTypes& st);
   JITTypeTag getJITTag() { return getJITTag(get_type_info().get_type()); }
 
  protected:
   SQLTypeInfo type_info;  // SQLTypeInfo of the return result of this expression
   bool contains_agg;
 
+  std::vector<cider::jitlib::JITValuePointer> nulls_;
+  std::vector<cider::jitlib::JITValuePointer> vals_;
```

Comment:
please use `expr_var_`, it contains data and null

Reply:
It's fine, I have refactored them in next PR.

Reply:
> please use `expr_var_`, it contains data and null

I tried it, but because we disabled assignment and copy construction,  will cause some errors. Still have to construct vector to pass. I think we can modify it after jiang refactor PR